src/app/context/theme.context.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import React, { Dispatch, ReactNode, SetStateAction, createContext, useContext, useMemo, useState } from 'react';
+import cn from 'classnames';
+
+interface TTheme {
```

Comment:
1. Why do you start the name of the interface with T and not with I?
2. Why do you need this `TTheme` if you’re using it just in one place? Would it be better to delete it and add this type to `TThemeContext`

Reply:
1) It was a type before, forgot to rename
2) Inattentiveness